### ISSUE TITLE
Interactive Content? Escaping the linear Page/Form/linear media paradigm.

### ISSUE THREAD

**Issue by brennanyoung** [w3c/wcag3#381]
I would greatly appreciate some clear recognition in WCAG 3 that not all in-browser content is oriented towards linear page/document/form abstractions. We produce medical simulation software (for training nurses and lifesavers) which runs in the browser, and I must say it has been an uphill struggle to find clear guidance which were not all about pages and forms. 

In particular, the browsable/operable distinction (which is central to many AT implementations) is poorly represented in WCAG for those who need to build anything with an interaction model based on something more rich than just form-filling and submission. The time-based media guidelines in earlier versions of WCAG seemed only vaguely applicable to content such as games and simulations. Time-based media is neither Prerecorded nor Live when scripted animations have their own logic, or include random elements, often with a huge variety of unpredictable output. Where do these kinds of multimedia fit in? How do we approach the construction of audio descriptions for something which is different for every user, every single time?

Or, say, if you have a cutaway view of some complex machine or organ, which allows you to 'open' layers and reveal parts within, is it more appropriate to say that the user 'browses' or 'operates' this content? You're not 'operating' the heart when you inspect a diagram of the valves and atria, surely?

Or, say, you have a lifesaving algorithm, explaining which steps should be taken depending on the symptoms you observe. How can you talk about a 'meaningful sequence', when there are multiple possible sequences through the algorithm, and the only truly meaningful sequence is the one which saves the life of the patient, yet WCAG says "only one meaningful sequence is sufficient" in this case. This particular piece of guidance is useless, but we can earn compliance by compromising the quality of our content to a potentially lethal degree.

I do understand the historical emphasis in guiding people in accessible *page* construction, and media players since so much of the web is about that, but I hope that other kinds of content, especially more fluid application-like interactions will get explicit attention in the new WCAG. 

I've been advised that there are ongoing efforts elsewhere to produce 'accessible game design' guidelines, which would be very useful for training simulations too, but where is the w3c in that effort? And what use are those separate efforts, if our games and simulations are nonetheless legally obliged to be WCAG compliant?
